graphene_federation/external.py

```diff
@@ -0,0 +1,26 @@
+from graphene import Schema
+
+
+def external(field):
+    """
+    Mark a field as external.
+    """
+    field._external = True
+    return field
+
+
+def get_external_fields(schema: Schema) -> dict:
+    """
+    Find all the extended types from the schema.
+    They can be easily distinguished from the other type as
+    the `@external` decorator adds a `_external` attribute to them.
+    """
+    external_fields = {}
+    for type_name, type_ in schema.graphql_schema.type_map.items():
+        if not hasattr(type_, "graphene_type"):
+            continue
+        for field in list(type_.graphene_type.__dict__):
```

Comment:
@erikwrede is there a better way to get fields from a Graphene type?

Reply:
`graphene_type._meta.fields` should usually work, `dict` is problematic as it is not limted to  graphene fi